helix-rest/src/main/java/org/apache/helix/rest/server/json/instance/StoppableCheck.java

```diff
@@ -78,4 +78,9 @@ public boolean isStoppable() {
   public List<String> getFailedChecks() {
     return failedChecks;
   }
+
+  public void add(StoppableCheck other) {
```

Comment:
2 comments,
1. I think we should call it "mergeFailures", or something similar, like appendFailures.
2. Can we merge to check results with the different categories? If not, shall we assert when trying to merge?

Reply:
1. I think `add(Stoppable)` is better. What I consider is like `list.addAll();`. 
`mergeFailures` should be `void mergeFailures(failures1, failures2);` which does not fit in this design. Similar to appendFailures.
2. Each stoppable is a different categories. They are strings. So the failures list is actually a list of strings of different categories: `[HELIX:abc, CUSTOM_PARTITION:efg, CUSTOM_INSTANCE: xyz]`

Reply:
I see. I misunderstood the content of the string. Maybe you can help to add some comments in the code to help the readers? That would be very helpful.

Regarding the name, "add" makes sense under this context, which I didn't understand before. Please keep it.